docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+database:
+  image: "sameersbn/mysql:latest"
+  environment:
+    DB_USER: staytus
+    DB_PASS: staytus
```

Comment:
This means we've now got a hardcoded database username/password, which is slightly messy security-wise. It'd be nice if we could randomly generate a password and share it between the two here. I'm not sure how to do that nicely in docker-compose mind.

The goal here hopefully is to having something people can actually take a throw up into production to get a status site really easily & safely, without carefully configuring everything themselves. Having a production site running with totally default standard passwords like this isn't good.

Doesn't necessarily need to block this, since it's not an urgent problem, but secure-by-default would be a good thing to aim for if there's an easy route there.